abd-clam/Cargo.toml

```diff
@@ -43,15 +43,17 @@ priority-queue = "1.3.2"
 # TODO: Break CHAODA out into an optional feature
 # automl = { version = "0.3.0", path = "../rust-automl" }
 # automl = { git = "https://github.com/nishaq503/rust-automl", branch = "updates/dependencies" }
+automl = "0.2.7"
 # Only used for normalizing cluster ratios for CHAODA
 libm = "0.2.7"
 statistical = "1.0.0"
 
 # Only used for sharded datasets
 # TODO: Break sharded datasets out into an optional feature
-# ndarray = { version = "0.15.6", features = ["rayon"] }
-# ndarray-npy = "0.8.1"
+ndarray = { version = "0.15.6", features = ["rayon"] }
+ndarray-npy = "0.8.1"
```

Comment:
Remove the `ndarray-npy` dependency and any of its uses in code. We will replace IO of arrays with an in-house solution